samples/csharp-language/expression-trees/ExpressionTreeBuildingSampleOne.cs

```diff
@@ -27,9 +28,8 @@ public override void Run()
             var ySquared = Expression.Multiply(yParameter, yParameter);
             var sum = Expression.Add(xSquared, ySquared);
 
-            Expression<Func<double, double>> sqrt = (x) => Math.Sqrt(x);
-            var methodCall = sqrt.Body as MethodCallExpression;
-            var distance = methodCall.Update(default(Expression), new List<Expression> { sum });
+            var sqrtMethod = typeof(Math).GetMethod("Sqrt", new[] { typeof(double) });
+            var distance = Expression.Call(sqrtMethod, sum);
```

Comment:
I like this code better, too :smile:

Unfortunately, it's not supported on .NET Core. That's why I went with the original (and more complicated) construct.

If you can back out this one code change (and the example code in the .md file) I'll merge.